docs/user/dql/functions.rst

```diff
@@ -1756,6 +1756,29 @@ Examples::
     +-----------------------------------+
 
 
+GET_FORMAT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a string value containing string format specifiers based on the input arguments.
+
+Argument type: TYPE, STRING
+TYPE must be one of the following tokens: [DATE, TIME, DATETIME, TIMESTAMP].
+STRING must be one of the following tokens: ["USA", "JIS", "ISO", "EUR", "INTERNAL"] (" can be replaced by ').
```

Comment:
only "USA" support for now, right?

Reply:
Ah yes. I will adjust the documentation for now and update it once the other formats are added.

Reply:
Fixed [here ](https://github.com/opensearch-project/sql/pull/1299/files#:~:text=os%3E%20select%20GET_FORMAT(DATE%2C%20%27USA%27)%3B)